File: Triple_de_Pitágoras.py
clicle = range(1, 500)
for c in clicle:
    for b in clicle:
        for a in clicle:
            # si "a" es mayor que todos
            if((a > b and a > c) and ((b**2+c**2) == a**2)):
                print('-----------------------------------------------------------')
                print('A=', a, " B=", b, " C=", c)
                print('''
                                      *
                                    ***
                        A=%s     *****       B=%r
                                *******
                              *********
                            ***********
                            
                                C=%d
                            ''' % ("{:<3}".format(a), b, c))
                print("B^2 + C^2 = A^2")
                print(b, "^2 + ", c, "^2 =", a, "^2")
                print(b**2, ' + ', c**2, " = ", a**2)
                print((b**2+c**2), " = ", a**2)

            # si "b" es mayor que todos
            elif((b > a and b > c) and ((a**2+c**2) == b**2)):
                print('-----------------------------------------------------------')
                print('A=', a, " B=", b, " C=", c)
                print('''
                            *
                            ***
                 A=%s      *****       B=%r
                            *******
                            *********
                            ***********
                            
                                C=%d
                            ''' % ("{:<3}".format(a), b, c))
                print("A^2 + C^2 = B^2")
                print(a, "^2 + ", c, "^2 =", b, "^2")
                print(a**2, ' + ', c**2, " = ", b**2)
                print((a**2+c**2), " = ", b**2)
            # si "c" es mayor que todos
            elif((c > a and c > b) and ((a**2+b**2) == c**2)):
                print('-----------------------------------------------------------')
                print('A=', a, " B=", b, " C=", c)
                print('''
                             
                             *
                            ***       
                A=%s     *******      B=%r
                        ***********
                      ***************
                    *******************
                            
                           C=%d
                            ''' % ("{:<3}".format(a), b, c))
                print("A^2 + B^2 = C^2")
                print(a, "^2 + ", b, "^2 =", c, "^2")
                print(a**2, ' + ', b**2, " = ", c**2)
                print((a**2+b**2), " = ", c**2)

File: Triple_de_Pitágoras.cpp
#include <iostream>
#include <cmath>
#include <iomanip>

using namespace std;

int main()
{

    for (int c = 1; c < 500; c++)
    {
        for (int b = 1; b < 500; b++)
        {
            for (int a = 1; a < 500; a++)
            {
                if ((a > b && a > c) && ((pow(b, 2) + pow(c, 2)) == pow(a, 2)))
                {
                    cout << "-----------------------------------------------------------\n";
                    cout << "A=" << a << " | B=" << b << " | C=" << c << "\n\n";
                    cout << "            *\n";
                    cout << "           **\n";
                    cout << "   A=" << setfill(' ') << setw(3) << a << "  ***  B=" << b << "\n";
                    cout << "         ****\n";
                    cout << "        *****\n";
                    cout << "       ******\n";
                    cout << "        C=" << c << "\n\n";
                    cout << "B^2 + C^2 = A^2\n";
                    cout << b << "^2 + " << c << "^2 =" << a << "^2\n";
                    cout << pow(b, 2) << " + " << pow(c, 2) << " = " << pow(a, 2) << "\n";
                    cout << (pow(b, 2) + pow(c, 2)) << " = " << pow(a, 2) << "\n";
                }

                // si "b" es mayor que todos
                if ((b > a && b > c) && ((pow(a, 2) + pow(c, 2)) == pow(b, 2)))
                {
                    cout << "-----------------------------------------------------------\n";
                    cout << "A=" << a << " | B=" << b << " | C=" << c << "\n\n";
                    cout << "          *\n";
                    cout << "          **\n";
                    cout << "   A=" << setfill(' ') << setw(3) << a << "  ***  B=" << b << "\n";
                    cout << "          ****\n";
                    cout << "          *****\n";
                    cout << "          ******\n";
                    cout << "           C=" << c << "\n\n";
                    cout << "A^2 + C^2 = B^2\n";
                    cout << a << "^2 + " << c << "^2 =" << b << "^2\n";
                    cout << pow(a, 2) << " + " << pow(c, 2) << " = " << pow(b, 2) << "\n";
                    cout << (pow(a, 2) + pow(c, 2)) << " = " << pow(b, 2) << "\n";
                }
                // si "c" es mayor que todos
                if ((c > a && c > b) && ((pow(a, 2) + pow(b, 2)) == pow(c, 2)))
                {
                    cout << "-----------------------------------------------------------\n";
                    cout << "A=" << a << " | B=" << b << " | C=" << c << "\n\n";
                    cout << "              *\n";
                    cout << "             ***\n";
                    cout << "    A=" << setfill(' ') << setw(3) << a << "  *******  B=" << b << "\n";
                    cout << "         ***********\n";
                    cout << "       ***************\n";
                    cout << "     *******************\n";
                    cout << "             C=" << c << "\n\n";
                    cout << "A^2 + B^2 = C^2\n";
                    cout << a << "^2 + " << b << "^2 =" << c << "^2\n";
                    cout << pow(a, 2) << " + " << pow(b, 2) << " = " << pow(c, 2) << "\n";
                    cout << (pow(a, 2) + pow(b, 2)) << " = " << pow(c, 2) << "\n";
                }
            }
        }
    }
}